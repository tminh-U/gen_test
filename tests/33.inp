6
17 17 10 18 6 7 
11
2 3
1 2
1 3
2 3
6 6
2 5
5 5
5 5
1 6
6 6
2 4
