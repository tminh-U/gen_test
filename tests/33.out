3
4
5
3
2
7
2
2
13
2
6
